releasenotes v4.6.1 (#54)
bisdn/bisdn-linux: v4.6.1-releasenotes.txt

BISDN Linux
v4.6.1 Release Notes
Date: 2022-06-08

! New in this release
- baseboxd: updated to version 1.11.5
- Linux kernel updated to version 5.15.44
- stfp and sftp-server were added in preparation of OpenSSH 9.0+ upgrades on host systems (which changes the default protocol of scp to SFTP)

! Bug fixes

> - baseboxd: a fix was added to properly handle default routes on data ports

Reviewer: We fixed the sfp leds for as4630-54pe.
We fixed the reported fan direction of the psus of as5385-54x.

Author: I could not find any commits regarding the latter issue about the PSUs.

Reviewer: > as5835-54x: fix reporting of psu fan direction

Author: Mention of the SFP LED fix has been added

Author: Added Fan PSU direction

! Changed behaviour

> - baseboxd/rofl-ofdpa: UDP rules were added so that DHCP packets are now forwarded to the CPU port by default. This enables using a data port as a management port, or allow it to receive configuration via DHCP

Reviewer: ```suggestion
- baseboxd/rofl-ofdpa: UDP rules were added so that DHCP packets are now forwarded to the CPU port by default. This allows receiving configuration via DHCP on front panel ports, or running a DHCP server on the switch.
```

BISDN Linux
v4.6.1 Release Notes
Date: 2022-06-08

! New in this release
- baseboxd: updated to version 1.11.5
- Linux kernel updated to version 5.15.44
- stfp and sftp-server were added in preparation of OpenSSH 9.0+ upgrades on host systems (which changes the default protocol of scp to SFTP)

! Bug fixes
- baseboxd: a fix was added to properly handle default routes on data ports

> - meta-ofdpa: fixed the SFP LEDs on the Edgecore Networks EPS202 (AS4630-54PE)

Reviewer: This has also a meta-open-network-linux part, though in the "high level" description it does not really matter where it was fixed, just what was fixed.

```suggestion
- Edgecore EPS202 (AS4630-54PE): fixed the SFP LEDs
```

> - meta-open-network-linux: Fixed reporting of PSU fan direction on the Edgecore Networks DCS201 (AS5835-54X)

Reviewer: ```suggestion
- Edgecore DCS201 (AS5835-54X): fixed reporting of PSU fan direction in onlpdump
```
